### PR TITLE
Add tags

### PR DESCRIPTION
**Pull request by croesus**
This isn't just adding tags. I thought the other commits in this PR had already been merged.

### CODE REVIEW

File: docs/metadata.rst [hunk 1]
```diff
@@ -106,6 +106,15 @@ geospatial and temporal ranges of entries within a data set. This is a complex s
...
 `here <https://www.w3.org/TR/vocab-dcat-2/#time-and-space>`_.
 
+We encourage use of the ``dcat:keyword`` list for data sets. These translate to "tags" in our web interface and are useful to group data sets around specific topics.
+
+.. code-block:: yaml
+
+  dcat:keyword: 
+    solar
+    electricity
+    retrofit
```
Comment (tomhukins) [inline docs/metadata.rst:116]: I'm fairly sure that in YAML each list item needs a `- ` prefix so this won't parse correctly.  However, YAML is a complicated format and I could easily be wrong.

Also, when I look at our [metadata parser's source code](https://github.com/icebreakerone/open-energy-python-infrastructure/blob/72eddd2d09a2f09b7383d4a9939c8fd035b56b2f/src/python/ib1/openenergy/support/metadata.py#L120) I wonder if this should be `dcat:keywords` (plural, not singular).

If you've tested this example and seen it work, then please ignore my concerns.

**Review by tomhukins — COMMENTED** [review]
This mostly looks good to me, but I'm unsure about the example YAML.  If my concern is misplaced, please consider this review an approval.

File: docs/metadata.rst (hunk 1 above)
Comment (croesus) [reply docs/metadata.rst]: Thanks @tomhukins. Good spot on the YAML - It passed the YAML parser in Sphinx which lulled me into a false sense of security, but it is incorrect for a YAML list. I've pushed a fix to this branch - please double-check.

The [correct DCAT property is keyword](https://www.w3.org/TR/vocab-dcat-3/#Property:resource_keyword) (singular). Confusingly the metadata parser parses `keyword` to its own property `keywords`. I can confirm the singular version has been fetched correctly to our index and converted to CKAN tags.

**Comment by tomhukins** [comment]
I approve of these changes but I can't find the approve button in GitHub's UI at the moment.

**Review by tomhukins — APPROVED** [review]

**Pull request merged**